utils/generate_types/test/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "strict": true,
     "target": "ES2019",
+    "lib": ["esnext", "dom", "DOM.Iterable"],
```

Comment:
This suggests that this change will break compilation for our users that do not have `esnext` target enabled.

Reply:
I don't think they'll compile them, and VS Code resolves it to @types/next:

<img width="528" alt="Screenshot 2023-10-24 at 9 15 59 AM" src="https://github.com/microsoft/playwright/assets/883973/fca490d0-38b2-4584-8fec-2ca071ed2e11">